packages/commercetools/composables/src/getters/checkoutShippingGetters.ts

```diff
@@ -0,0 +1,45 @@
+import { CheckoutShippingGetters } from '@vue-storefront/core';
+import { Address } from './../types/GraphQL';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getAddresses = (shipping: any, criteria?: Record<string, any>) => [];
```

Comment:
I think we should get rid of `getAddresses` and `getMethods` wdyt?

Reply:
why?

Reply:
`getAddresses` and `isAddressDefault` are related to `useUserShipping`
`getMethods` is related to `useCheckoutShippingMethod`

Reply:
I think some getters could overlap here and thats ok

Reply:
DX-wise it makes sense to have all under 1 fn

Reply:
nope, you don't have to be user to read the address from checkout

Reply:
considering shipping object on the checkout page, you have only one address:
```ts
export const getAddress = (shipping: ) => Address

```